src/docs/get-started/install/_chromeos-android-sdk-setup.md

```diff
@@ -0,0 +1,54 @@
+## Android setup (without Android Studio)
+
+### Install Java
+
+```terminal
+$ sudo apt update
+$ sudo apt install default-jre
+$ sudo apt install default-jdk
+```
+
+### Install the Android SDK's
+
+Download the [Android SDK tools]({{site.android-dev}})/studio/#downloads and 
+select the “Command Line Tools only” option.
+
+Drag and drop the downloaded zip into your Linux Files folder through the 
+Chrome OS Files app. This moves it to the home directory, notated as $TOOLS_PATH 
+going forward (`~/`)
+
+Unzip the tools and then add it to your path
+
+```terminal
+$ unzip ~/sdk-tools-linux*
+$ export PATH="$PATH:$TOOLS_PATH/tools/bin"
+```
+
+Download the SDK packages using the sdkmanager tool (version numbers here are 
```

Comment:
It might be worth adding a quick note here:
"Navigate to `~/` (or wherever you'd like to keep SDK packages) and download the following SDK packages using the `sdkmanager tool..."